distribution/lib/Standard/Base/0.0.0-dev/src/System.enso

```diff
@@ -50,6 +50,10 @@ nano_time = @Builtin_Method "System.nano_time"
 os : Text
 os = @Builtin_Method "System.os"
 
+## Check if the operating system is UNIX.
+is_unix : Boolean
```

Comment:
Our Platform knows only 3 OSes plus "unknown". I don't think we mean to support any other than these 3, but still, checking `is_unix === is_linux || is_mac` just seemed a bit wrong to me since there are other UNIX systems and we have this way of checking it which is more robust for any possible (although indeed rather extremely unlikely) expansions.

Reply:
Sure, I will move it.